src/esp/io/JsonEspTypes.h

```diff
@@ -71,6 +90,10 @@ inline JsonGenericValue toJsonValue(const esp::assets::AssetInfo& x,
   addMember(obj, "virtualUnitToMeters", x.virtualUnitToMeters, allocator);
   addMember(obj, "requiresLighting", x.requiresLighting, allocator);
   addMember(obj, "splitInstanceMesh", x.splitInstanceMesh, allocator);
+  if (x.overridePhongMaterial != Cr::Containers::NullOpt) {
```

Comment:
You can directly serialize the Corrade::Optional! Thanks to some template magic in src/esp/io/JsonMagnumTypes.h.

`addMember(obj, "overridePhongMaterial", x.overridePhongMaterial, allocator);` should just work, and similar for readMember.